src/main/resources/application.yaml

```diff
@@ -76,6 +79,10 @@ folio:
         concurrency: ${KAFKA_INSTANCE_AUTHORITY_STATS_CONSUMER_CONCURRENCY:1}
         topic-pattern: (${folio.environment}\.)(.*\.)links\.instance-authority-stats
         group-id: ${folio.environment}-mod-entities-links-stats-group
+      instance-authority-update-failure:
+        concurrency: ${KAFKA_INSTANCE_AUTHORITY_UPDATE_FAILURE_CONSUMER_CONCURRENCY:1}
+        topic-pattern: (${folio.environment}\.)(.*\.)links\.instance-authority-update-failure
+        group-id: ${folio.environment}-mod-entities-links-authorities-group
```

Comment:
We don't need a new topic, we already have `instance-authority-stats` topic that handles failures